vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/VimOptionGroupBase.kt

```diff
@@ -16,7 +16,9 @@ import com.maddyhome.idea.vim.vimscript.model.datatypes.VimDataType
 public abstract class VimOptionGroupBase : VimOptionGroup {
   private val globalOptionsAccessor = GlobalOptions()
   private val globalValues = mutableMapOf<String, VimDataType>()
+  private val globalParsedValues = mutableMapOf<String, Any>()
```

Comment:
This makes me questioning how do we work in a multiconcurrent environment. And I'm nore really sure if we support multiconcurrency, but what abut changing this to some concurrent map "just in case"?

Reply:
I'd rather do it deliberately because we know it's being accessed concurrently, rather than just in case. And I think if this could be accessed concurrently, then so would a lot of other data structures we have. Or to put it another way, I'm not using this map any differently to the other maps in the class. We should probably look at this more deeply to see what our concurrency is.